src/integrationTest/java/uk/gov/hmcts/darts/task/service/OutboundAudioDeleterProcessorTest.java

```diff
@@ -68,6 +68,16 @@ void setUp() {
         when(currentTimeHelper.currentOffsetDateTime()).thenReturn(OffsetDateTime.of(2023, 10, 27, 22, 0, 0, 0, ZoneOffset.UTC));
     }
 
+    @BeforeEach
```

Comment:
Never seen this before so unsure whether its totally necessary but if it is could possibly be added as a juni5 rule.

Reply:
This mimics the open-in-view sessions strategy thats enabled on our spring deployments.  Means we don't have to introduce transactions into production code to make integration tests pass.  I will be good to attempt to role it out to all integration tests but I tried and wasn't easy.  For this PR just leaving it in this test. Will attempt to expand it in separate ticket.  It should probably be used going forward though.  I think "@ExtendWith" has replaced Rules in junit5.